src/services/user/services/AdminUsers.js

```diff
@@ -120,9 +120,21 @@ class AdminUsers {
 
 	async remove(id, params) {
 		const { _ids } = params.query;
+		const currentUser = await getCurrentUserInfo(params.account.userId);
+
 		if (id) {
+			const userToRemove = await getCurrentUserInfo(id);
+			if (currentUser.schoolId.toString() !== userToRemove.schoolId.toString()) {
```

Comment:
please use the compare helper (`const { equal: equalIds } = require('../../../helper/compare').ObjectId;`)